portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +48,8 @@ function next() {
   }
 }
 
-// function that changes slideshow's picture to the previous
-// picture when the user presses the previous button
+/* function that changes slideshow's picture to the previous
```

Comment:
Sorry for not being clear, it should be

```
/**
 * function ...
 * ...
 */
```

Reply:
Also the comment should generally starts with a verb.

So instead of "a function that changes..." you can directly use "changes..."

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments has more details (you can ignore the parts about @param and @return, but other style guides still apply here).